packages/frontend/src/app/app.scss

```diff
@@ -30,6 +30,14 @@ ion-menu.cookingToolbarOpen {
   font-size: 40px;
 }
 
+.medium-icon {
+  font-size: 2em;
+}
+
+.eye-icon {
+  vertical-align: middle;
+}
+
```

Comment:
[non-blocking] Would be nice for these styles not to sit in the global app.scss, but it looks like directives can't apply stylesheets directly. I do see some mention of directives supporting a `host` property here, though https://stackoverflow.com/a/35915497/21691840